configure.ml

```diff
@@ -1156,7 +1156,7 @@ let write_makefile f =
   pr "CAMLHLIB=%S\n\n" camllib;
   pr "# Caml link command and Caml make top command\n";
   pr "# Caml flags\n";
-  pr "CAMLFLAGS=-rectypes %s %s %s\n" coq_warn_flags coq_annotate_flag coq_safe_string;
+  pr "CAMLFLAGS=%s\n" caml_flags;
```

Comment:
Umm, why is this stuff printed twice by configure and `envvars.ml` ?

Reply:
Not sure I understand the question.

FTR:
./configure generates config/coq_config.ml and config/Makefile.
envars is a wrapper around coq_config.ml letting the user override some values by setting appropriate environment variables.